src/test/java/com/contentgrid/gateway/runtime/session/PartitionedWebSessionManagerTest.java

```diff
@@ -0,0 +1,141 @@
+package com.contentgrid.gateway.runtime.session;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.security.test.web.reactive.server.SecurityMockServerConfigurers.mockOAuth2Login;
+import static org.springframework.security.test.web.reactive.server.SecurityMockServerConfigurers.springSecurity;
+
+import com.contentgrid.gateway.GatewayApplication;
+import com.contentgrid.gateway.runtime.session.PartitionedWebSessionManagerTest.TestController;
+import com.contentgrid.gateway.test.util.LoggingExchangeFilterFunction;
+import java.time.Duration;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.Import;
+import org.springframework.stereotype.Controller;
+import org.springframework.test.web.reactive.server.WebTestClient;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.server.WebSession;
+
+@Slf4j
+@SpringBootTest(
+        classes = GatewayApplication.class,
+        properties = {
+                "contentgrid.gateway.runtime-platform.enabled=true"
+        })
+@Import(TestController.class)
+class PartitionedWebSessionManagerTest {
+
+    private static final String SESSION_COOKIE_NAME = "SESSION";
+
+    @Controller
+    static class TestController {
+
+        @GetMapping("/test")
+        @ResponseBody
+        String test(WebSession session) {
+            // explicitly start session to trigger session cookie
+            session.start();
+
+            return "OK";
+        }
+    }
+
+    @Autowired
+    ApplicationContext context;
+
+    private WebTestClient client;
+
+    @BeforeEach
+    public void setup() {
+        this.client = WebTestClient
+                .bindToApplicationContext(this.context)
+                .apply(springSecurity())
+                .configureClient()
+                .filter(new LoggingExchangeFilterFunction(log::info))
+                .responseTimeout(Duration.ofHours(1))
+                .build();
+    }
+
+    @Test
+    void sessionCookie_attributes() {
+
+        var sessionCookie = client
+                .mutateWith(mockOAuth2Login())
+                .get()
+                .uri("https://app.userapps.contentegrid.app/test")
```

Comment:
```suggestion
                .uri("https://app.userapps.contentgrid.app/test")
```

That's a typo. Not that it really matters a lot for the tests, but still.